client/components/checklist/style.scss

```diff
@@ -188,83 +184,79 @@
 	.spinner {
 		display: none;
 		position: absolute;
-		top: 12px;
+		top: 17px;
 		left: 24px;
 	}
 
-	&-warning-background {
+	.checklist__task-warning-background {
 		display: block;
 		position: absolute;
-		top: 18px;
-		left: 28px;
+		top: -1px;
+		left: 0;
 		width: 18px;
 		height: 18px;
 		border-radius: 16px;
 		background: var( --color-white );
 		cursor: pointer;
 	}
 
-	&-primary {
-		display: flex;
-		flex-direction: column;
-		flex: 1 1 90%;
+	.checklist__task-title {
+		text-align: left;
 	}
 
-	&-secondary {
-		display: flex;
-		flex: 1 1 10%;
-		align-items: flex-start;
-		padding-left: 10px;
+	.checklist__task-title-button.button {
+		position: relative;
+		display: block;
+		padding: 16px $task-right-padding 4px $task-left-padding;
+		width: 100%;
+		text-align: left;
+		color: var( --color-neutral-dark );
+		font-weight: 600;
+		font-size: 16px;
+		border-radius: 0;
 
-		@include breakpoint( '<480px' ) {
-			padding-left: 0;
-			align-items: center;
+		.checklist__toggle-affordance {
+			position: absolute;
+			right: 12px;
+			top: 16px;
+			fill: var( --color-neutral-light );
+			transition: $chevron-animation;
 		}
-	}
 
-	&-title-link.button {
-		padding: 0;
-		color: var( --color-primary );
-		font-weight: 400;
-		font-size: 16px;
+		&:hover {
+			cursor: pointer;
+
+			.checklist__toggle-affordance {
+				fill: var( --color-primary );
+			}
+		}
 	}
 
-	&-description {
-		word-break: break-word;
-		font-size: 14px;
+	.checklist__task-content {
+		padding: 0 $task-right-padding 16px $task-left-padding;
 	}
 
-	&-completed-description {
+	.checklist__task-description {
 		word-break: break-word;
 		font-size: 14px;
-		display: none;
+		margin-bottom: 16px;
 	}
 
-	&-duration {
-		font-size: 12px;
-		color: var( --color-text-subtle );
-
-		.checklist__task-primary & {
-			display: inline-block;
-		}
-
-		.checklist__task-secondary & {
-			display: none;
-		}
+	.checklist__task-action-duration-wrapper {
+		display: flex;
+		flex-direction: column-reverse;
+	}
 
-		@include breakpoint( '<480px' ) {
-			.checklist__task-primary & {
-				display: none;
-			}
+	.checklist__task-duration {
+		font-size: 14px;
+		color: var( --color-neutral-dark );
+	}
 
-			.checklist__task-secondary & {
-				display: inline-block;
-				margin-left: 1.5em;
-			}
-		}
+	.checklist__task-action-wrapper {
```

Comment:
Do we need the extra wrapper `<div />`? Can't we just assign the bottom margin to `.checklist__task-action`?

Reply:
The wrapper `divs` are here to help with hiding/showing the details and also for some visual styling.